Makefile

```diff
@@ -240,4 +241,3 @@ server:
 ### Clean ###
 clean:
 	rm -rf build/
-    
```

Comment:
Hm?

Reply:
A single trailing line with some whitespace on it.
Did it serve any purpose?

Reply:
I guess it should have been a single trailing line without whitespace.

Reply:
Other files in the repo don't have trailing newlines either. Do we want a trailing newline here?

Reply:
Huh, I thought they all did. They all should! Don't worry about the other files but let's retain one here.